package/network/services/wireguard/Makefile

```diff
@@ -5,26 +5,43 @@
 #
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
+#
+# When Linux 5.4 support is removed, kmod-wireguard should be moved to
+# package/kernel/linux/modules/netsupport.mk and this package removed.
 
 include $(TOPDIR)/rules.mk
 include $(INCLUDE_DIR)/kernel.mk
 
 PKG_NAME:=wireguard
 
-PKG_VERSION:=1.0.20201112
-PKG_RELEASE:=1
+PKG_CONFIG_DEPENDS:=CONFIG_LINUX_5_10
```

Comment:
This doesn't make sense.
Why not add it as a kernel module and rename the existing to wireguard-oot?